addons/website/__manifest__.py

```diff
@@ -144,6 +144,7 @@
         ],
         'web._assets_primary_variables': [
             'website/static/src/scss/primary_variables.scss',
+            ('prepend', 'website/static/src/scss/website.navbar.variables.scss'),
```

Comment:
Normally the point of having an assets splits in multiple part (variable, secondary, bootstrap, etc) is to avoid the need of prepend/append/etc: you should normally be able to just "add at the end of the parent module files". So my question is: why the need of prepend here?

Reply:
Long story short, we decided to use `primary_variables.scss` for global variables only (eg. brand colors).
Component-related variables will be moved into each component folder, in a scss file with `.variable.scss` suffix. 

A glob fetching all these files  [has been added](https://github.com/odoo/odoo/pull/82434/files#diff-66d417d1828879bbc2124ddc118b47030ea7c30fb93e660c5231365420002214R304) in `web` manifest.
```
'web._assets_primary_variables': [
    'web/static/src/legacy/scss/primary_variables.scss',
    'web/static/src/**/**/*.variables.scss', ⬅︎
],
```
It will probably cause a bit of file juggling between files, but the approach is more consistent with the modular way we're pushing forward with owl.

cc. @aab-odoo

Reply:
The reason was that, by default, navabar links use `$o-font-size-base` that's 13px in the backend, but 12px in the frontend.

I'm now realising that this will override the backend value too, I didn't notice that because both aim for same value, 13px.

I'll "force" website navbar links in CSS for now... it will not be pretty like a variable change, but as you said all of this will be removed soon